packages/opencensus-core/src/trace/model/types.ts

```diff
@@ -232,6 +232,8 @@ export interface TraceOptions {
   spanContext?: SpanContext;
   /** Span kind */
   kind?: SpanKind;
+  /** Determines the sampling rate. Ranges from 0.0 to 1.0 */
+  samplingRate?: number;
```

Comment:
Should this be a `Sampler` instance per the specs?

Reply:
I am happy to change here, but the global sampler takes the same input https://github.com/census-instrumentation/opencensus-node/blob/f98ec08248d7e962738278473d762ab449eabe49/packages/opencensus-core/src/trace/config/types.ts#L35 To keep things consistent I use ```number```. WDYT?

Reply:
Yeah, I'm OK with that, but could you create an issue to track changing it in both places to bring it closer to the specs?

Reply:
sure.